src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,54 @@
+package lotto.domain;
+
+import lotto.util.NumberGenerator;
+import lotto.util.RandomNumberGenerator;
+
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+import static lotto.constant.ErrorMessage.INVALID_PRICE;
+
+public class Lottos {
+    public static final int DEFAULT_VALUE = 0;
+    public static final int INCREASE_UNIT = 1;
+
+    private final List<Lotto> lottos = new ArrayList<>();
+    private final NumberGenerator generator = new RandomNumberGenerator();
+
+    public Lottos(int payment) {
+        validate(payment);
+
+        for (int i = 0; i < payment / Lotto.PRICE; i++) {
+            lottos.add(new Lotto(generator));
+        }
+    }
+
+    public Lottos(List<Lotto> lottos) {
+        this.lottos.addAll(lottos);
+    }
+
+    private void validate(int payment) {
+        if (payment % Lotto.PRICE != 0) {
+            throw new IllegalArgumentException(INVALID_PRICE.getMessage());
+        }
+    }
+
+    public Map<Rank, Integer> getRankCount(WinningNumbers winningNumbers) {
+        Map<Rank, Integer> rankCount = new EnumMap<>(Rank.class);
+        lottos.forEach(lotto ->
+            rankCount.put(
+                winningNumbers.getRank(lotto),
+                rankCount.getOrDefault(winningNumbers.getRank(lotto), DEFAULT_VALUE) + INCREASE_UNIT));
+        return rankCount;
+    }
```

Comment:
아래와 같이 변경해볼 수도 있을 것 같아요. 😃

추가적으로 네이밍에 대해서 전체적으로 조금 고민해보시면 좋을 것 같습니다. 
가급적 `getXXX`, `setXXX`와 같은 네이밍 대신에 이 메서드의 행위를 드러내는 네이밍을 사용하도록 변경해보시면 좋을 것 같습니다. 💪

```suggestion
    public Map<Rank, Long> getRankCount(WinningNumbers winningNumbers) {
        return lottos.stream()
            .map(winningNumbers::getRank)
            .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
    }
```

Reply:
이렇게 하는 경우, 사용됐던 상수 2개(DEFAULT_VALUE, INCREASE_UNIT)을 지울 수 있게 됐네요.
혹시 이 외에도 장점이 있을까요?

```java
    public int getTicketCount() {
        return lottos.size();
    }
```

❓위와 같은 코드의 경우, 단순히 정보를 가져오기 때문에 get이라는 메서드명을 사용했습니다. 
이런 경우, 로키는 어떤 메서드명을 주로 사용하시나요?

Reply:
사실 이 메서드도 포워딩을 할 뿐이지, 사실상 getter와 다르지 않다고 생각해요. 😃
따라서 저였어도 훌라랑 비슷하게 지었을 것 같아요. 

저는 개인적으로 이미 자바 언어 혹은 라이브러리등에서 쓰고있는 네이밍을 많이 참고하는 편인데요. 

`getLottoSize()` 혹은 이미 Lottos 라는 클래스명 덕분에 `size()` 라고 네이밍해도  **lottos.size()** 같이 호출되게되어 의미를 잘 이해할 수 있을 것 같네요.